dist/index.html

```diff
@@ -14,40 +14,22 @@ <h1>LeaderBoard</h1>
           <div class="score-div">
             <div class="score-title">
               <h2>Recent Scores</h2>
-              <button type="button">Refresh</button>
+              <button type="button" id="refreshBtn">Refresh</button>
             </div>
-            <table class="score-Table">
-                <tr>
-                    <td>Name:100</td>
-                </tr>
-                <tr>
-                    <td>Name:20</td>
-                </tr>
-                <tr>
-                    <td>Name:50</td>
-                </tr>
-                <tr>
-                    <td>Name:78</td>
-                </tr>
-                <tr>
-                    <td>Name:125</td>
-                </tr>
-                <tr>
-                    <td>Name:77</td>
-                </tr>
-                <tr>
-                    <td>Name:42</td>
-                </tr>
-            </table>
+            <ul class="scores" id="data-container">
+                
+            </ul>
           </div>
       </div>
 
       <div class="sub-main2">
         <h1>Add Your Score</h1>
 
-            <input type="text" placeholder="Your Name"> <br>
-            <input type="text" placeholder="Your Name"> <br>
-            <button type="submit">Submit</button>
```

Comment:
- [ ] Great job with the styling. please change the score placeholder from `your name` to `your score/add score`
![Screenshot (143)_LI](https://user-images.githubusercontent.com/65298048/174032845-10f7986d-0fa5-4b52-a09e-0b9cdef1d5c6.jpg)